src/SeqViz/SeqViz.tsx

```diff
@@ -265,10 +263,9 @@ export default class SeqViz extends React.Component<SeqVizProps, any> {
     let { annotations } = this.state;
 
     // part is either from a file/accession, or each prop was set
-    seq = seq || part.seq || "";
-    // @ts-expect-error ts-migrate(2345) FIXME: Argument of type 'string | undefined' is not assig... Remove this comment to see the full error message
-    if (getSeqType(seq) === "dna") {
-      compSeq = compSeq || part.compSeq || dnaComplement(seq).compSeq;
+    const _seq: string = seq || part.seq || "";
```

Comment:
the `_prefix` identifiers are not normal in TS, recommended against. Let's remove or find a diff way to avoid duplicate identifier names https://google.github.io/styleguide/tsguide.html#identifiers